<launch>

    <node pkg="nodelet" type="nodelet" name="radar_manager_left" args="manager" output="screen"/>

    <node pkg="nodelet" type="nodelet" name="rm_radar_left"
          args="load rm_radar/Lidar radar_manager_left" output="screen">
        <rosparam file="$(find rm_radar)/config/camera_left.yaml" command="load" ns="left_camera_proc_config"/>
        <param name="camera_name" value="left_camera_proc"/>
    </node>

    <node pkg="nodelet" type="nodelet" name="radar_manager_right" args="manager" output="screen"/>

    <node pkg="nodelet" type="nodelet" name="rm_radar_right"
          args="load rm_radar/Lidar radar_manager_right" output="screen">
        <rosparam file="$(find rm_radar)/config/camera_right.yaml" command="load" ns="right_camera_proc_config"/>
        <param name="camera_name" value="right_camera_proc"/>
    </node>

     <node pkg="nodelet" type="nodelet" name="radar_manager_UI" args="manager" output="screen"/>
     <node pkg="nodelet" type="nodelet" name="UI" args="load rm_radar/UI radar_manager_UI" output="screen">
            <rosparam file="$(find rm_radar)/config/map.yaml" command="load" ns="map_config"/>
            <param name="is_red" value="false"/>
     </node>

    <include file="$(find rm_detector)/launch/armor_detector.launch"/>
    <include file="$(find livox_ros_driver)/launch/livox_lidar.launch"/>

    <node name="rviz" pkg="rviz" type="rviz" respawn="true" args="-d $(find rm_radar)/config/UI.rviz"/>
</launch>
